### PR TITLE
Only allow configured parameters in POST requests

### PR DESCRIPTION
**Pull request by leleuj**

### CODE REVIEW

**Comment by mmoayyed** [comment]
:+1:

File: src/main/java/org/jasig/cas/security/RequestParameterPolicyEnforcementFilter.java [hunk 1]
```diff
@@ -122,6 +121,11 @@
     public static final String LOGGER_HANDLER_CLASS_NAME = "loggerHandlerClassName";
 
     /**
+     * The name of the optional Filter init-param specifying what request parameters ought to be send via POST requests only.
+     */
+    public static final String ONLY_POST_PARAMETERS = "onlyPostParameters";
```
Comment (mmoayyed) [inline src/main/java/org/jasig/cas/security/RequestParameterPolicyEnforcementFilter.java]: One extra comment: I think `postOnlyParameters` might be a better fit for the param name.

**Comment by mmoayyed** [comment]
@leleuj lets go ahead and merge this one.

**Comment by leleuj** [comment]
OK

**Pull request merged**